.github/workflows/stale-branches.yml

```diff
@@ -13,9 +13,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout repository
-        uses: actions/checkout@7884fcad6b5d53d10323aee724dc68d8b9096a2e # v2
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # ratchet:actions/checkout@v3
```

Comment:
I think the comment is wrong and this should be v2? Matches the pre-diff file, and the sha doesn't match the one for other actions/checkout@v3 in other files in this PR.

Reply:
Good catch, thanks.